cmd/kube-controller-manager/app/certificates.go

```diff
@@ -82,6 +83,15 @@ func startCertificateSigningRequestSigningController(ctx context.Context, contro
 			return nil, false, fmt.Errorf("failed to start kubernetes.io/kube-apiserver-client certificate controller: %v", err)
 		}
 		go kubeAPIServerClientSigner.Run(ctx, 5)
+
+		pcrClient := controllerContext.ClientBuilder.ClientOrDie("podcertificate-controller")
+		pcrInformer := controllerContext.InformerFactory.Certificates().V1alpha1().PodCertificateRequests()
+
+		kubeAPIServerPodClientSigner, err := signer.NewPodClientCSRSigningController(ctx, pcrClient, pcrInformer, clock.RealClock{}, kubeAPIServerSignerCertFile, kubeAPIServerSignerKeyFile)
```

Comment:
Need a different top level controller for this.  An end user should be able to independently stop signing for kubelets and pods.

Reply:
This was removed from this PR so we can get the basic functionality landed in 1.33